src/hooks/abrt-hook-ccpp.c

```diff
@@ -623,6 +623,11 @@ int main(int argc, char** argv)
                         (long)pid, executable);
     }
 
+    const char *signame = NULL;
+    const int fatal_signal = signal_is_fatal(signal_no, &signame);
+
+    log_notice("Processing '%s' got signal SIG%s", executable, signame);
```

Comment:
If `signal_no` is not a fatal one, `signame` is `NULL` (`signal_is_fatal()` function does not translate all signal numbers to human readable strings), so you will see message like this in the logs:
`Processing '/usr/bin/foo' got signal SIG(null)`

How about to move `reason` from line 848 to above this line and use `reason`  for the logging purpose too?

`reason` doesn't contain full path but I think we don't need to included this information in the message. This is how dmesg line looks like when a process gets SEGV:
`[ 5059.103443] will_segfault[7418]: segfault at 0 ip 000055851b95fa00 sp 00007ffd2c631120 error 4 in will_segfault[55851b95f000+1000]`
The message is rather strange, but does not contain the full path. Also If I run a poor quality code written by myself, I will see very long log messages, because those binaries are stored in a deep sub-directory structure in my home directory.

Even `systemd-coredump` includes only the base name:
`Oct 14 07:43:14 photon systemd-coredump[7588]: Process 7587 (will_segfault) of user 18601 dumped core.`

Speaking of `systemd-coredump` , no one is complaining about spamming system logs, so be progressive and use `log` instead of `log_notice` (perhaps you can log it as ERROR, to see a nice, red colored message in journald). I wanted to do that but I didn't have time to come up with a good plan for better logging (I am burden with D-Bus API v2, CVEs and bureaucracy, you know), so I have been keeping status quo since I silenced `abrt-hook-ccpp` on William Jon McCann's request.